client/src/views/HomeView.tsx

```diff
@@ -259,6 +265,11 @@ const HomeView = () => {
                 onChange={changeName}
                 value={name}
               />
+              <Tooltip title={!fixBruteAppearance ? t('lockBruteAppearance') : t('unlockBruteAppearance')}>
```

Comment:
Ternary operators should have their positive outcome first, and avoid using `!` in their main condition